func main()
	try
	catch 1.0
	end try
end func
